app/models/manageiq/providers/vmware/network_manager/refresh_parser.rb

```diff
@@ -28,36 +28,90 @@ def ems_inv_to_hashes
 
     private
 
-    def get_networks
-      # fetch org VDC networks
-      @inv[:networks] = @org.networks
+    def log_header
```

Comment:
move to the bottom, I guess, because it's a util function that should not interfere with the logic